force-app/main/default/aura/HH_AddressMgr/HH_AddressMgr.cmp

```diff
@@ -52,7 +53,7 @@
     <!-- CHANGE ADDRESS POPUP -->
     <aura:if isTrue="{!v.showChangeAddressPopup}">
         <div id="changeAddressPopup" class="slds-modal slds-fade-in-open" aria-hidden="false"
-             role="dialog" aria-labelledby="searchPopupLabel">
+             role="dialog" aria-labelledby="searchPopupLabel" tabindex="-1">
```

Comment:
I've chosen to set the focus on the modal div itself instead of any elements inside of the modal since programmatic focus cannot be explicitly set on aura components in the `lightning` namespace. This also will announce the dialogue and the elements contained within including the modal close button.